xhr.bs

```diff
@@ -317,22 +317,22 @@ from {{XMLHttpRequestEventTarget}} as attributes:
    <td>{{XMLHttpRequest/loadstart}}
```

Comment:
```suggestion
   <td>{{XMLHttpRequestEventTarget/loadstart}}
```
not sure how I managed to miss it :)

it's a bit of a shame there isn't an easy way to CI detect this

Reply:
Yeah, agreed. Fortunately we don't change dfns often.